src/cli/format.ts

```diff
@@ -76,7 +76,7 @@ export async function peprFormat(validateOnly: boolean) {
 
       return !hasFailure;
     } catch (e) {
-      console.error(e.message);
+      console.error(`Error formatting module: ${e.message}`);
```

Comment:
```suggestion
      console.error(e, `Error formatting module`);
```

Reply:
All messages updated to display complete `e`